packages/contracts/src/ubiquistick/UbiquiStick.sol

```diff
@@ -33,16 +33,16 @@ contract UbiquiStick is
     address public minter;
 
     string private _tokenURI;
-    uint256 private constant STANDARD_TYPE = 0;
+    uint256 private constant _STANDARD_TYPE = 0;
```

Comment:
@AnakinSkywalkeer  is this refactor `STANDARD_TYPE => _STANDARD_TYPE` connected with libs update?

Reply:
> @AnakinSkywalkeer is this refactor `STANDARD_TYPE => _STANDARD_TYPE` connected with libs update?

it was not really but like editor was throwing errors about putting underscore as its a private and constant we can remove them if we want

Reply:
I think it's ok keep them with the `_` prefix